src/cascadia/TerminalSettingsEditor/ObjectModel/GlobalSettings.idl

```diff
@@ -36,7 +36,7 @@ namespace Microsoft.Terminal.Settings.Model
         Boolean WarnAboutLargePaste;
         Boolean WarnAboutMultiLinePaste;
         String LaunchPosition;
-        // TODO CARLOS: fix JsonConverters for these unique types
+        // fix JsonConverters for these unique types

```

Comment:
shouldn't this still have a real todo somewhere

Reply:
That's in code that's getting deleted but removed it anyways.